test/e2e/framework/providers/gce/.import-restrictions

```diff
@@ -0,0 +1,23 @@
+# This E2E framework sub-package is currently allowed to use arbitrary
+# dependencies except of k/k/pkg, therefore we need to override the
+# restrictions from the parent .import-restrictions file.
+#
+# At some point it may become useful to also check this package's
+# dependencies more careful.
+rules:
+  - selectorRegexp: "^k8s[.]io/kubernetes/pkg"
+    allowedPrefixes:
+    - k8s.io/kubernetes/pkg/api/legacyscheme
```

Comment:
Maintaining these lists will be problematic. Let's start doing that once we know that we want to limit usage of k/k/pkg in this package, i.e. for now don't add this file.

Reply:
I agree about this one, but would leave k8s.io/kubernetes/pkg/api/v1/pod if it's the only one dependency.

Reply:
done